manifests/init.pp

```diff
@@ -246,6 +246,9 @@
 # $dhcp_nameservers::           DHCP nameservers, comma-separated
 #                               type:String
 #
+# $dhcp_pxeserver::             DHCP default pxeserver
```

Comment:
A better description would be valuable here, e.g. `DHCP "next-server" value, defaults otherwise to IP of dhcp_interface`